src/ResourceManagement/Resource/Microsoft.Azure.Management.V2.Resource/Core/CreatableResources.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Management.V2.Resource.Core.CollectionActions;
+using Microsoft.Azure.Management.V2.Resource.Core.ResourceActions;
+using System.Threading;
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Management.V2.Resource.Core
+{
+    /// <summary>
+    /// Base class for creatable resource collection, i.e.those where the member of the collection is of Resource
+    /// type <see cref="IResource"> and are creatable.
+    /// (Internal use only)
+    /// </summary>
```

Comment:
please remove this line of comment. If this class should be internal only it should not declare itself public :)